homework-submission/react-hyf/react-hyf-client/src/HomeworkWeek3.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import Name from './Name';
+import Info from './Info';
+import ProfilePicture from './ProfilePicture';
+import Back from './Back';
+
+class HomeworkWeek3 extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      apiData: [],
+      randomPerson: {},
+      picture: '',
+    };
+  }
+
+  componentDidMount = e => {
+    axios
+      .get('https://uinames.com/api/?amount=10')
+      .then(response => {
+        this.setState({ apiData: response.data });
```

Comment:
Destructering data as function parameter would be nice here

```js
.then({ data } => {
```

Reply:
yes of course 
thanks a lot man!